src/shadydom.js

```diff
@@ -82,9 +84,23 @@ if (utils.settings.inUse) {
     // access that requires Shadow DOM behavior to be proxied via `ShadyDOM.wrap`.
     'noPatch': utils.settings.noPatch,
     'nativeMethods': nativeMethods,
-    'nativeTree': nativeTree
+    'nativeTree': nativeTree,
+    'upgrade': (fragment, host, options) => {
```

Comment:
`upgrade` just sounds too much like the CE concept... can we just call it `attachDom`?

Reply:
Changed.